### ISSUE TITLE
Problems trying to replace the OFFICE365 that I purchased in Sept 2020 but was blocked from using.

### ISSUE THREAD

**Issue by RHL-CPA** [MicrosoftDocs/OfficeDocs-Exchange#2512]
[Enter feedback here]
I am 83 years old and thought bu following your instructions this morning that I would finally be recognised by you. I was bocked when I purchased Office 365 by YOU saying I had to change my email account. I paid for the Office 365 but you blocked me from using it. I cancelled my credit card to protect me from having to pay for something that I cant use. I tried to return by getting a newer version and was scammed $3800 by a Microsoft tech that came on my computer and nd charged through Paypal as a CLOUD or something. And I stopped the process a month ago. I would like to get the OFFICE365  with 6 terabytes for my family, but tried to get the trial version to make sure it worked for ME. I am a retired CPA and cant even use my cell phone anymore, but you will send tour code only to it. 416 618 3961  What am I supposed to do? Ray H Luff CA retired in 1964 and tired of only being able to play games on my computers. My house Phone ir 705 999 5499, ut you don’t recognize it.

HELP ME.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 40c92567-1479-8a19-9ee6-cc3563ed5bfc
* Version Independent ID: 313638dc-b9dc-9e58-50a9-b46245c8e35e
* Content: [Fix email delivery issues for error code 550 5.1.1 through 5.1.20 in Exchange Online](https://docs.microsoft.com/en-us/exchange/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-1-1-through-5-1-20-in-exchange-online)
* Content Source: [Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-1-1-through-5-1-20-in-exchange-online.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-1-1-through-5-1-20-in-exchange-online.md)
* Service: **o365-administration**
* Technology: ****
* GitHub Login: @msdmaguire
* Microsoft Alias: **dmaguire**

**Comment by RHL-CPA** [comment]
[Enter feedback here]
I am 83 years old and thought bu following your instructions this morning that I would finally be recognised by you. I was bocked when I purchased Office 365 by YOU saying I had to change my email account. I paid for the Office 365 but you blocked me from using it. I cancelled my credit card to protect me from having to pay for something that I cant use. I tried to return by getting a newer version and was scammed $3800 by a Microsoft tech that came on my computer and nd charged through Paypal as a CLOUD or something. And I stopped the process a month ago. I would like to get the OFFICE365 with 6 terabytes for my family, but tried to get the trial version to make sure it worked for ME. I am a retired CPA and cant even use my cell phone anymore, but you will send tour code only to it. 416 618 3961 What am I supposed to do? Ray H Luff CA retired in 1964 and tired of only being able to play games on my computers. My house Phone ir 705 999 5499, ut you don’t recognize it.

HELP ME.

**Comment by RHL-CPA** [comment]
[Enter feedback here]
I am 83 years old and thought bu following your instructions this morning that I would finally be recognised by you. I was bocked when I purchased Office 365 by YOU saying I had to change my email account. I paid for the Office 365 but you blocked me from using it. I cancelled my credit card to protect me from having to pay for something that I cant use. I tried to return by getting a newer version and was scammed $3800 by a Microsoft tech that came on my computer and nd charged through Paypal as a CLOUD or something. And I stopped the process a month ago. I would like to get the OFFICE365 with 6 terabytes for my family, but tried to get the trial version to make sure it worked for ME. I am a retired CPA and cant even use my cell phone anymore, but you will send tour code only to it. 416 618 3961 What am I supposed to do? Ray H Luff CA retired in 1964 and tired of only being able to play games on my computers. My house Phone ir 705 999 5499, ut you don’t recognize it.

HELP ME.
